### ISSUE TITLE
Add short section to the overview chapter covering the sample code

### ISSUE THREAD

**Issue by hollasch** [RayTracing/raytracing.github.io#833]
In particular, we should stress that our code is a toy implementation, isn't a recommended C++ implementation, leaves a lot of features on the table, and so forth.

**Comment by trevordblack** [comment]
This belongs in both the README and in the index.html, maybe even in the opening introduction of book 1

**Comment by hollasch** [comment]
Yeah, I was thinking about the introduction to book 1. That should be sufficient to inform the reader of the underlying philosophy. That said, we should add some lightweight guidance as well in the CONTRIBUTING doc about the intent of the code and our approach to things like engineering quality, performance, utility, and the like. I think that with those two, we don't need to put this up front (or not more than a sentence) in the README or on the web site.

**Comment by hollasch** [comment]
We also need to be careful not to discourage too much simple improvements to the code that still meet our requirements for simplicity and exposition.

**Comment by trevordblack** [comment]
> Sees stack of issues in v4

so it's one of those "Do as I say, not as I do" kind of deals?

**Comment by hollasch** [comment]
If you have an issue with one or more of our work items, it would probably be more productive to have those discussions in the associated issues.

**Comment by trevordblack** [comment]
It was a bad joke

**Comment by hollasch** [comment]
No, Trevor. It's no parking in the _yellow_ zone. The white zone is for loading and unloading of passengers only.
